core/files/css/base.php

```diff
@@ -327,7 +327,9 @@ public function add_control_rules( array $control, array $controls_stack, callab
 
 		$stylesheet = $this->get_stylesheet();
 
-		$control = apply_filters( 'elementor/files/css/selectors', $control, $value ?? [] );
+		$post_id = method_exists( $this, 'get_post_id' ) ? $this->get_post_id() : false;
+
+		$control = apply_filters( 'elementor/files/css/selectors', $control, $value ?? [], $post_id );
```

Comment:
A parent class shouldn't know about children implementations (`method_exists`...)
Please create a method `get_control` that a child can override and add his logic.

Reply:
Done